Docs/install/install_host.rst

```diff
@@ -59,33 +73,38 @@ Install the basic pre-requisite packages as follows:
+1.2 If you have multiple Python versions installed, set the default version.
 
 .. code-block:: bash
 
     update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.10 1
     update-alternatives --set python3 /usr/bin/python3.10
 
-Install GPU packages
-~~~~~~~~~~~~~~~~~~~~~
 
-**NOTE:**
+**2. Install the CUDA toolkit (optional).**
+
+.. note::
+
+    The GPU toolkit is required only for GPU variants of AIMET.
+    
+The released AIMET GPU packages have been tested with the following CUDA toolkit versions:
 
-#. Do this section ONLY for the GPU variants.
-#. The released AIMET GPU packages *were tested* with the following CUDA toolkit versions:
-    #. PyTorch 2.1 GPU variant: `CUDA Toolkit 12.1.0 <https://developer.nvidia.com/cuda-12-1-0-download-archive>`_
-    #. PyTorch 1.13 GPU variant: `CUDA Toolkit 11.7.1 <https://developer.nvidia.com/cuda-11-7-1-download-archive>`_
-    #. TensorFlow GPU variant: `CUDA Toolkit 11.8.0 <https://developer.nvidia.com/cuda-11-8-0-download-archive>`_
-    #. ONNX GPU variant: `CUDA Toolkit 11.7.1 <https://developer.nvidia.com/cuda-11-7-1-download-archive>`_
-#. The instructions in the sub-sections below correspond to our tested versions above. Visit this page https://developer.nvidia.com/cuda-toolkit-archive to obtain the correct version of the CUDA toolkit for your environment.
+- PyTorch 2.1 GPU variant: `CUDA Toolkit 12.1.0 <https://developer.nvidia.com/cuda-12-1-0-download-archive>`_
+- PyTorch 1.13 GPU variant: `CUDA Toolkit 11.7.1 <https://developer.nvidia.com/cuda-11-7-1-download-archive>`_
+- TensorFlow GPU variant: `CUDA Toolkit 11.8.0 <https://developer.nvidia.com/cuda-11-8-0-download-archive>`_
+- ONNX GPU variant: `CUDA Toolkit 11.7.1 <https://developer.nvidia.com/cuda-11-7-1-download-archive>`_
 
-Install GPU packages for PyTorch 2.1 or PyTorch 1.13 or ONNX or TensorFlow
-==========================================================================
+2.1 Visit the CUDA Toolkit link above for the verison corresponding to your AIMET GPU package and download the tested version of the CUDA toolkit for your environment.
 
-**NOTE:**
+All versions of the CUDA toolkit are also listed at https://developer.nvidia.com/cuda-toolkit-archive.
 
-#. Visit this page https://developer.nvidia.com/cuda-12-1-0-download-archive or https://developer.nvidia.com/cuda-11-8-0-download-archive or https://developer.nvidia.com/cuda-11-7-1-download-archive to obtain the exact and up-to-date installation instructions for your environment.
-#. Please do not execute the final command "sudo apt-get install cuda" provided in aforementioned NVIDIA documentation links.
+.. note::
+
+    In the next step, do not execute the final command, **sudo apt-get install cuda**, in the install instructions.
+
+2.2 Follow the command-line instructions on the download page to install the CUDA toolkit, but do *not* execute the final command, **sudo apt-get install cuda**.
```

Comment:
Let's do this like this to make it a bit more clear:
`on the download page` --> `on the developer.nvidia.com download page`

Reply:
Changed.